dask/async.py

```diff
@@ -429,6 +417,10 @@ def get_async(apply_async, num_workers, dsk, result, cache=None,
     """
     assert queue
 
+    if callbacks is None:
+        callbacks = _globals['callbacks']
+    callbacks = setup_callbacks(callbacks)
```

Comment:
Could a callback object return a `do_nothing` function if the corresponding callback does not exist?